### PR TITLE
Remove Tokenizer default models

### PR DESCRIPTION
**Pull request by andrewfrench**
- Removed default models from Tokenizers
- Added MODEL_PREFIX_TO_MAX_TOKENS mapping to relevant Tokenizers
- Audited existing model, max token mappings

### CODE REVIEW

**Comment by andrewfrench** [comment]
Pyright checks are failing due to unrelated issues in test files.

File: griptape/chunkers/base_chunker.py [hunk 1]
```diff
@@ -14,9 +14,7 @@ class BaseChunker(ABC):
     separators: list[ChunkSeparator] = field(
         default=Factory(lambda self: self.DEFAULT_SEPARATORS, takes_self=True), kw_only=True
     )
-    tokenizer: BaseTokenizer = field(
-        default=Factory(lambda: OpenAiTokenizer(model=OpenAiTokenizer.DEFAULT_OPENAI_GPT_3_CHAT_MODEL)), kw_only=True
-    )
+    tokenizer: BaseTokenizer = field(default=Factory(lambda: OpenAiTokenizer(model="gpt-3.5-turbo")), kw_only=True)
```
Comment (cjkindel) [inline griptape/chunkers/base_chunker.py:17]: What is the value of shifting from an global enum to a local string? I understand removing defaults to require defining a model, but in this case we are just replacing a enum defined to be "gpt-3.5-turbo" with the string "gpt-3.5-turbo".

Comment (collindutter) [reply griptape/chunkers/base_chunker.py:17]: I'm actually kind of inclined to remove from the use of `OpenAiTokenizer` entirely. Chunkers are low level enough where this should be something another class passes in.

**Comment by collindutter** [comment]
Implemented in https://github.com/griptape-ai/griptape/pull/690

**Pull request closed without merging**